src/importers/safeInCloudXmlImporter.ts

```diff
@@ -39,7 +42,7 @@ export class SafeInCloudXmlImporter extends BaseImporter implements Importer {
         });
 
         Array.from(doc.querySelectorAll('database > card')).forEach(cardEl => {
-            if (cardEl.getAttribute('template') === 'true') {
+            if (cardEl.getAttribute('template') === 'true' || cardEl.getAttribute('deleted') === 'true') {
```

Comment:
Just to be sure, is there an easy way in SafeInCloud to un-delete items so if someone notices something missing they can recover? Sorry to use you as a go-between here ;)

Reply:
> Just to be sure, is there an easy way in SafeInCloud to un-delete items so if someone notices something missing they can recover? Sorry to use you as a go-between here ;)

Yes, you can just right-click and press "restore". It's possible to restore multiple deleted entries or all via CTRL+A